src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-discovery-modal/iscsi-target-discovery-modal.component.html

```diff
@@ -17,7 +17,8 @@
             <input id="user"
                    class="form-control"
                    formControlName="user"
-                   type="text">
+                   type="text"
+                   autocomplete="off">
```

Comment:
Should we also add this to the `mutul_user`?

Reply:
> Should we also add this to the `mutul_user`?

Makes sense.